src/components/Editor/Editor.js

```diff
@@ -28,7 +28,7 @@ export default function Editor() {
       </div>
       <div className="form-control">
         <label>Alignment</label>
-        <div className="radio-group">
+        <div className="radio-group" onChange={(e) => { setAlign(e.target.value); }}>
```

Comment:
Excellent work Mary! I believe if we add a value={Align}, it would allow for the defaults to be set, as you have in your useState hooks. If not, we may have to switch our onChange function to be within the input element.